src/service/command/meme/kenjou.ts

```diff
@@ -8,8 +10,7 @@ export const kenjou: MemeTemplate<
   (typeof positionalKeys)[number]
 > = {
   commandNames: ['kenjou'],
-  description:
-    '[健常者エミュレーター](https://healthy-person-emulator.memo.wiki/)の構文ジェネレーター。\n健常者エミュレーターWikiにありそうなタイトルを指定すればうまくいきます。',
+  description: `[健常者エミュレーターWiki](${sourceLink})の構文ジェネレーター。\n健常者エミュレーターWikiにありそうなタイトルを指定すればうまくいきます。`,
```

Comment:
```suggestion
  description: `[健常者エミュレーター事例集](${sourceLink})の構文ジェネレーター。\n健常者エミュレーター事例集にありそうなタイトルを指定すればうまくいきます。`,
```
サイト名も Wiki から事例集に変わっているのでこうしたほうがよいかと

Reply:
同じ言葉が続いたので、 https://github.com/approvers/OreOreBot2/pull/761/commits/19e06af0faa0c0b2c35b4d7fce2aada2e7c51afb で修正しました。